dnf-behave-tests/dnf/repo-json.feature

```diff
@@ -0,0 +1,151 @@
+            "repo_file_path":"\/tmp\/dnf_ci_installroot_*\/etc\/yum.repos.d\/dnf-ci-fedora.repo",
+            "base_url":[
+              "file:\/\/\/*\/dnf-behave-tests\/fixtures\/repos\/dnf-ci-fedora"
+            ],
+            "metalink":"",
+            "mirrorlist":"",
+            "gpg_key":[],
+            "repo_gpgcheck":false,
+            "gpgcheck":false,
+            "available_pkgs":289,
+            "pkgs":289,
+            "size":"*",
+            "content_tags":[],
+            "distro_tags":[],
+            "revision":"1550000000",
+            "max_timestamp":"*"
+          },
+          {
+            "id":"dnf-ci-thirdparty-updates",
+            "name":"dnf-ci-thirdparty-updates test repository",
+            "is_enabled":true,
+            "priority":99,
+            "cost":1000,
+            "type":"available",
+            "exclude_pkgs":[],
+            "include_pkgs":[],
+            "timestamp":"*",
+            "metadata_expire":172800,
+            "skip_if_unavailable":false,
+            "repo_file_path":"\/tmp\/dnf_ci_installroot_*\/etc\/yum.repos.d\/dnf-ci-thirdparty-updates.repo",
+            "base_url":[
+              "file:\/\/\/*\/dnf-behave-tests\/fixtures\/repos\/dnf-ci-thirdparty-updates"
+            ],
+            "metalink":"",
+            "mirrorlist":"",
+            "gpg_key":[],
+            "repo_gpgcheck":false,
+            "gpgcheck":false,
+            "available_pkgs":6,
+            "pkgs":6,
+            "size":"*",
+            "content_tags":[],
+            "distro_tags":[],
+            "revision":"1550000000",
+            "max_timestamp":"*"
+            }
+        ]
+        """
+
+
+Scenario: Repo list without arguments --json
+   When I execute dnf with args "repo list --json"
+   Then the exit code is 0
+    And stdout json matches
+    """
+    [
+      {
+        "id":"dnf-ci-fedora",
+        "name":"dnf-ci-fedora test repository",
+        "is_enabled":true
+      },
+      {
+        "id":"dnf-ci-thirdparty-updates",
+        "name":"dnf-ci-thirdparty-updates test repository",
+        "is_enabled":true
+      }
+    ]
+    """
+
+
+Scenario: Repo list without arguments --json --all
+   When I execute dnf with args "repo list --json --all"
+   Then the exit code is 0
+    And stdout json matches
+    """
+    [
+      {
+        "id":"dnf-ci-fedora-updates",
+        "name":"dnf-ci-fedora-updates test repository",
+        "is_enabled":false
+      },
+      {
+        "id":"dnf-ci-fedora",
+        "name":"dnf-ci-fedora test repository",
+        "is_enabled":true
+      },
+      {
+        "id":"dnf-ci-thirdparty-updates",
+        "name":"dnf-ci-thirdparty-updates test repository",
+        "is_enabled":true
+      },
+      {
+        "id":"dnf-ci-thirdparty",
+        "name":"dnf-ci-thirdparty test repository",
+        "is_enabled":false
+      }
+    ]
+    """
+
+
+Scenario: Repo list without arguments --json --refresh --quiet
```

Comment:
Is there a reason for this scenario? It seems to be essentially the same as the `Repo list without arguments --json`, and I'm not sure why the --refresh and --quiet options are important. Could you clarify their significance?

Reply:
It was meant to test that `<REPOSYNC>` doesn't get into the stdout.
It was really confusing and wrong.

The `--quiet` is not needed because its implicitly set by `--json` and I have changed it to `repo info` because `repo list` doesn't need metadata. 
I have updated it.
 
Thank you for catching that.